src/test/java/org/fundacionjala/coding/Jose/EANTest.java

```diff
@@ -0,0 +1,28 @@
+package org.fundacionjala.coding.Jose;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by JoseTorrez on 5/30/2017.
+ */
+public class EANTest {
+    /**
+     *
+     */
+    @Test
+    public void rightNumber() {
+       EAN eanValidation = new EAN();
+       assertTrue(eanValidation.validate("4003301018398"));
+    }
+    /**
+     *
+      */
+    @Test
+   public void wrongNumber() {
+               EAN eanValidation = new EAN();
```

Comment:
You are repeating this code in all your tests
Please put them in @before method or convert it to a static method.